doc/sphinx/dev_instruct.rst

```diff
@@ -48,7 +46,7 @@ where ``$CONDA_ENV_DIR`` is the path you used to install the Conda environments.
 
    c. After installation, run ``% conda clean --a`` to clear cache.
 
-   d. Don't forget to update ``src/conda/env_python3_base.yml`` accordingly.
+   d. *DO NOT* forget to update ``src/conda/env_python3_base.yml`` accordingly.
```

Comment:
How are we going to test that changes to this base environment do not break other PODs?

Reply:
@jkrasting In the short term--probably by applying Tom's script to the branch to ensure that all PODs build and run on GFDL systems (analysis, and possibly the FMS Skylake and AMD dev boxes), and that no new errors are generated wrt the develop branch. 

@yihungkuo I assume that you have run the package with and without the additions from this PR on your own system(s). Please add a note about that you verified that your changes cause no new issues in the PR description (include the OS and Python version you used). 

Down the line, testing can be automated by adding a yaml file that launches a version of the run script to build the PODs, eventually adding lightweight run test cases that ideally work on Travis and/or GH actions when users supply stripped-down data. If data or processor requirements are too large for Travis/Actions, then run tests will likely be limited to hand-launched internal testing until we can incorporate containerized workflows on our systems.

Reply:
@wrongkindofdoctor Will do. I think this is a great opportunity for testing the PR template!

Reply:
That sounds good @wrongkindofdoctor.  That will catch any major issues.

Yes, good idea @yihungkuo - let's kick the tires on the PR template.